bin/setup-faxws.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+
+echo "Setting up the Fax server"
+
+## 3. Clone
+git clone https://dennis_warren@bitbucket.org/openoscar/faxws.git
+
+## 4. Checkout
+git checkout origin/release
+
+# 5. current MySQL pass and username
+echo "Setting Fax WS database passwords... "
+
+read -p "MySQL database username? " username
+read -p "MySQL database password? " passwd
+
+perl -pi -e "s/{user}/$username/"  ./src/main/webapp/META-INF/context.xml
```

Comment:
Can these be passed into a properties file? Or have to be compiled in?

Reply:
They could be in a properties file, but then I would have to change the code :)  It's a lot tidier just to compile this into the war.